### DISCUSSION TITLE
Srceen size of xmal layout

### DISCUSSION THREAD

**Discussion by WeiZhou-dev** [Satish-Mutyala/T24S2T1EXTG12#8] [Q&A]
Hi team, how do we handle varying screen sizes? Should we stick with fixed sizes for a simpler design?

**Comment by JeongseopYun** [comment]
Hello Wei, I have used 13.3 desktops since that was the preset when we were learning C#.

**Reply by Satish-Mutyala** [reply]
Yes, standard windows 13.3 should align with  the ITWorks dev.
